tests/openassetio-python/CMakeLists.txt

```diff
@@ -173,12 +173,18 @@ openassetio_add_pytest_target(
 # - openassetio-python-venv
 openassetio_add_pytest_target(
     openassetio.internal.resources-examples-pytest
-    "Running pytest for examples"
+    "Running pytest for Manager examples"
```

Comment:
[question] Should we bother with any tests under `examples` now?  There's only SAM, and that's so bare bones it doesn't really test much of anything.  Maybe SAM should just disappear altogether, for now?

Reply:
I'd be happy with that, maybe migrate it to a "template manager plugin"?

Reply:
Happy to do this in a follow up PR?